webapp/src/webapp/features/runbooks/helpers.cljs

```diff
@@ -0,0 +1,12 @@
+(ns webapp.features.runbooks.helpers
+  (:require
+   [clojure.string :as cs]))
+
+(defn extract-repo-name
+  "Extract just the repository name from a full repository path.
+   Example: 'github.com/hoophq/runbooks' -> 'runbooks'"
+  [repo-path]
+  (if (string? repo-path)
```

Comment:
the repo-path could not be a string 🤔

Reply:
This function will receive the repo path as a string, but if it isn't, what will it be, and why is it returning? it looks like the repo name when not string but the repo-name is string as well right?